src/workerd/api/r2-bucket.h

```diff
@@ -230,6 +246,16 @@ class R2Bucket: public jsg::Object {
   jsg::Promise<kj::Maybe<jsg::Ref<HeadResult>>> put(jsg::Lock& js,
       kj::String key, kj::Maybe<R2PutValue> value, jsg::Optional<PutOptions> options,
       const jsg::TypeHandler<jsg::Ref<R2Error>>& errorType);
+  jsg::Promise<jsg::Ref<R2MultipartUpload>> createMultipartUpload(jsg::Lock& js,
+      kj::String key, jsg::Optional<MultipartOptions> options,
+      const jsg::TypeHandler<jsg::Ref<R2Error>>& errorType);
+  jsg::Promise<UploadedPart> uploadPart(jsg::Lock& js,
+      kj::String key, kj::String uploadId, int partNumber, R2PutValue value,
+      const jsg::TypeHandler<jsg::Ref<R2Error>>& errorType);
+  jsg::Promise<void> abortMultipartUpload(jsg::Lock& js,
+      kj::String key, kj::String uploadId, const jsg::TypeHandler<jsg::Ref<R2Error>>& errorType);
+  jsg::Promise<jsg::Ref<R2Bucket::HeadResult>> completeMultipartUpload(jsg::Lock& js,
+      kj::String key, kj::String uploadId, kj::Array<UploadedPart> uploadedParts, const jsg::TypeHandler<jsg::Ref<R2Error>>& errorType);
```

Comment:
Nit: odd line wrapping here too

Reply:
Hmm, is there a style guide somewhere? I tried to follow the formatting of the lines above somewhat, but I agree that it looks weird. Would you prefer 1 line per argument in case all args push the line over 100 chars, or do you just want to see the `jsg::Lock& js` on another line (or 2 if necessary) with the rest of the arguments there?

Reply:
That's what I prefer but I think wrapping anything longer than 100 chars is ok